eos-update-flatpak-repos

```diff
@@ -467,6 +467,556 @@ FLATPAKS_TO_MIGRATE = [
         'new-branch': 'stable',
         'old-origin': 'eos-apps',
         'new-origin': 'flathub'
+    },
+    # migrate 61 legacy eos-apps to flathub (T23856)
+    {
+        'name': 'com.arduino.App',
+        'new-name', 'cc.arduino.arduinoide',
```

Comment:
The reason I requested review on #237 before reviewing this PR is that every migration dictionary added here has the same syntax error, namely a comma rather than a colon after `'new-name'`.

Reply:
Lulz. Yeah. I've not tried this yet. Didn't want to unleash it on any system until we had the X-Flatpak-RenamedFrom ducks in a row.

Reply:
Thats better... the new syntax checker likes it too.